package.json

```diff
@@ -8,7 +8,9 @@
     "test:watch": "jest --watch",
     "build": "webpack -p ",
```

Comment:
@Philipotieno just remove the space after the command

Reply:
Ok, it's done @koechkevin